README.md

```diff
@@ -2,7 +2,7 @@
 
 Crystal bindings to the excellent [LSQUIC](https://github.com/litespeedtech/lsquic) library.
 
-Releases track lsquic's versioning starting with `v2.18.1`.
+Releases track lsquic's versioning starting with `v2.23.1`.
```

Comment:
```suggestion
Releases track lsquic's versioning starting with `v2.18.1`.
```
This is about the version scheme changing with `v2.18.1`.

Reply:
ugh, I just grepped and replaced, sorry